### PR TITLE
Remove github.com/gobuffalo/flect and just correct the pluralization manually

### PR DESCRIPTION
**Pull request by enmand**

### CODE REVIEW

**Comment by jbowes** [comment]
This looks good to me; less code, one less dependency. Was there some case that the flect using code didn't cover, that a test could be written for?

File: idtype/idtype.go [hunk 1]
```diff
@@ -120,7 +117,12 @@ func (t Type) Mutable() bool {
 // Collection returns the name for a collection of these types
 func (t Type) Collection() string {
 	defn := getDefn(t)
-	return flect.Pluralize(defn.name)
+	switch {
+	case strings.HasSuffix(defn.name, "access"):
```
Comment (tim-speed) [inline idtype/idtype.go]: could maybe even reduce to suffix `ss` ? or `ess` ?

**Comment by enmand** [comment]
It's covered(ish) in the `access` case -- but the problem that for `flect.Pluralize` it needs either some hard-defined rules (translates these rules to these words: https://github.com/gobuffalo/flect/blob/master/plural_rules.go#L23) or some suffix rules (if the string ends in X, translate the ending to Y: https://github.com/gobuffalo/flect/blob/master/pluralize.go#L38). Before `LoadInflections` was used to load in `access` -> `access` plural inflection but that didn't add the suffix check. `flect.AddPlural` for adding rules appends the rule to the end, and iterates in order, so it see the "ss" rule first, and translates it (maybe correctly?) to `accesses`.

I'll update the tests to have some prefix to make sure the case we're changing some rules about pluralization it works

**Comment by enmand** [comment]
> That's a big change for go.sum, is that right??
I did `make mod-tidy` after removing the dep too, maybe I shouldn't have?

**Comment by tim-speed** [comment]
sorry didn't mean to close

**Pull request merged**